MAUI/Autocomplete/Searching-Filtering.md

```diff
@@ -171,6 +171,276 @@ autocomplete.TextSearchMode = AutocompleteTextSearchMode.Contains;
+
+{% tabs %}
+{% highlight XAML %}
+ <ContentPage.BindingContext>
+    <local:CityViewModel>
+    </local:CityViewModel>
+</ContentPage.BindingContext>
+  <editors:SfAutocomplete 
+    DisplayMemberPath="CityName"
+    ItemsSource="{Binding Cities}">
+            <editors:SfAutocomplete.FilterBehavior>
+                <local:CityFilteringBehavior/>
+            </editors:SfAutocomplete.FilterBehavior>
+  </editors:SfAutocomplete>
+
+{% endhighlight %}
+{% endtabs %}
+
+The following image demonstrates how to display cities in the drop-down based on the country name entered in the `Autocomplete` control.
+
+![.NET MAUI AutoComplete filter the items based on custom filtering logic.](Images/Searching_Filtering/Custom_filtering.png)
+
+### Choose default item to select
+
+When searching, the first item in the drop-down will be highlighted by default. Using the `SearchBehavior` property, you can customize the default highlighting behavior by using your custom selection logic to select the items based on your search criteria. The default value of `SearchBehavior` is `null`. 
+
+**Step 1:** Create a class that derives from the `IAutocompleteSearchBehavior` interface.
+
+{% tabs %}
+{% highlight C# %}
+
+/// <summary>
+/// Represents a custom searching behavior for `Autocomplete` control. 
+/// </summary>
+
+public class CapitalCitySearchingBehavior : Syncfusion.Maui.Inputs.IAutocompleteSearchBehavior
+
+{
+
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+**Step 2:** Then, implement the `GetHighlightIndex` method of the `IAutocompleteSearchBehavior` interface to initially select any item from the filtered list in the `Autocomplete` control drop-down. The `GetHighlightIndex` method contains the following arguments.
+
+* `source` - The owner of the search behavior, which holds information about `ItemsSource` and so on.
+* `searchInfo` - Contains information about the filtered items based on the text entered in `Autocomplete` control. You can select item from the drop-down list using this list. 
+
+The following code demonstrates initially how to select an item from the filtered list of drop-down in the `Autocomplete` control. When entering the country's name, the capital cities will be selected.
+{% tabs %}
+
+{% highlight C# %}
+
+public class CapitalCitySearchingBehavior : Syncfusion.Maui.Inputs.IAutocompleteSearchBehavior
+{
+    public int GetHighlightIndex(SfAutocomplete source, AutocompleteSearchInfo searchInfo)
+    {
+
+        var filteredCapitals = from CityInfo cityInfo in searchInfo.FilteredItems
+                               where cityInfo.IsCapital
+                               select searchInfo.FilteredItems.IndexOf(cityInfo);
+        if (filteredCapitals.Count() > 0)
+            return filteredCapitals.FirstOrDefault();
+
+        return 0;
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+**Step 3:** Applying custom searching to `Autocomplete` control by using the `SearchBehavior` property. 
```

Comment:
**Step 3:** Apply the custom searching to the `Autocomplete` control by using the `SearchBehavior` property.